docs/source/agent-framework/driver-framework/home-assistant/HomeAssistantDriver.rst

```diff
@@ -72,10 +88,15 @@ Your registry file should contain one device, with the ability to add attributes
        }
    ]
 
+.. note::
+
+When using a single registry file to represent a logical group of multiple physical entities, make sure the 
+"Volttron Point Name" is unique within a single registry file. For example, if a registry file contains entities with id  'light.instance1' and 'light.instance2' the entry for the attribute brightness for these two light instances could have "Volttron Point Name" as 'light1/brightness' and 'light2/brightness' respectively. This would ensure that data is posted to unique topic names and brightness data from light1 is not overwritten by light2 or viceversa.
```

Comment:
My driver currently uses the volttron point name to extract that specific attribute from the home assistant data. I can change it to allow for any name as long as brightness is incorporated but for now that's how it is.